File: rb2zfl/fixtures/x1_crossmethod.rb
class ToolController
  def run_cmd(c)
    system(c)                       # param -> shell (summary)
  end
  def index
    cmd = params[:cmd]
    run_cmd(cmd)                    # EXPECT: REFUTED (cross-method summary)
    run_cmd("safe")                 # EXPECT: clean
  end
end

File: cs2zfl/fixtures/x1_crossmethod.cs
using System.Diagnostics;
public class FController : Controller {
  void RunCmd(string c) { Process.Start("cmd", "/c " + c); }  // param -> shell (summary)
  public IActionResult Do() {
    var cmd = Request.Query["cmd"];
    RunCmd(cmd);                                 // EXPECT: REFUTED (cross-method summary)
    return View();
  }
}
